package/test/test.js

```diff
@@ -34,6 +38,10 @@ fetch('./test/mode.txt')
                 },
             });
 
+            editor.onDidChangeModelContent(() => {
```

Comment:
This is just for testing so not a big deal, but is this debounced?

Reply:
You are totally correct.
I will add debounce for that.